### DISCUSSION TITLE
Firefox extension still on 1.22 version

### DISCUSSION THREAD

**Discussion by kmacmillan25** [tprouvot/Salesforce-Inspector-reloaded#444] [General]
Hello, is it possible to update the Firefox extension to the new 1.23 release please? Thank you!

**Comment by tprouvot** [comment]
New version submitted to the store ! Apologies for the delay, I'll automate the release in the next version so that there won't be any delay 🚀

**Reply by kmacmillan25** [reply]
Thank you @tprouvot !
